CONTRIBUTING.md

```diff
@@ -43,6 +43,14 @@ Don't forget to add your new article to the `SUMMARY.md` to let it be rendered t
 
 Please make `Draft Pull requests` early so we can follow your progress and can give early feedback (see the following section).
 
+## Style guide
+
+- Follow the book [style guide](https://github.com/rust-lang/book/blob/master/style-guide.md).
+- Follow [RFC 1574](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#appendix-a-full-conventions-text). TL;DR:
```

Comment:
```suggestion
- Follow [RFC 1574](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#appendix-a-full-conventions-text).
- tl;dr
```
Would this be better? I don't see that is looks better but both seemed weird.

Reply:
It looks good to me in the same line as it is now. But maybe it is better to write `Tl;dr` instead of `TL;DR`, right?